vllm/engine/llm_engine.py

```diff
@@ -1156,6 +1163,10 @@ def _process_model_outputs(self,
             seq_group.maybe_set_first_token_time(now)
             if not seq_group.is_prefill():
                 seq_group.set_last_token_time(now)
+                stats_snapshot = self._get_stats(scheduler_outputs, outputs,
```

Comment:
nit: create a function to include get_stats then set inband_stats logic, since it repeats.